setup/sql/ilDBTemplate.php

```diff
@@ -33947,7 +33947,6 @@ function setupILIASDatabase()
 	)
 	,"insert_time" => array (
 		"notnull" => true
-		,"default" => "NOW()"
```

Comment:
this should be deleted in ilias3.sql and the db-step where it's introduced, too.